.swiftpm/xcode/xcshareddata/xcschemes/CombineExt.xcscheme

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you should not commit this file.

Reply:
add this to gitignore:
```
# Ignore all schemes located in swiftpm
**/.swiftpm/**/xcschemes/
```